vhdl_lang/src/ast.rs

```diff
@@ -1375,6 +1378,14 @@ pub struct ConfigurationSpecification {
     pub vunit_bind_inds: Vec<VUnitBindingIndication>,
 }
 
+/// LRM 7.4 Disconnection specification
+#[derive(PartialEq, Debug, Clone)]
+pub struct DisconnectionSpecification {
+    pub ident: Option<WithDecl<Ident>>,
```

Comment:
The way I would solve the ident problem is twofold.
First, declare an enum
```rust
pub enum DisconnectSignalSpecification {
    All,
    Others,
    Ident(WithDecl<Ident>)
}
```

And then change the disconnect specification ident to be an instance of that enum, i.e.,
```rust
pub struct DisconnectionSpecification {
    pub ident : DisconnectSignalSpecification,
    // ...
}
```

Then, to allow for multiple idents, just emit multiple specification, i.e.,
```vhdl
disconnect a, b, c : some_type after 10 ns;
```
would declare three `DisconnectSpecification`s, each with a different ident but with the same auxiliary information (i.e., the subtype and the time expression stays the same). This is also the way it's done for, for example, signal declarations.

Reply:
Done. It seems to be working.